### ISSUE TITLE
Feature Request - Reset Zoom Factor to Default on Zoom Exit

### ISSUE THREAD

**Issue by TheVisionaryOne** [Keksuccino/JustZoom#34]
Although this is a fantastic zoom mod, there is one feature I find it lacking compared to other zoom mods I've used in the past. Usually, when I exit zoom, the zoom factor is reset to the default zoom factor (let's say, 100%). If I've previously used my scroll wheel to zoom in to 200%, I exit zoom, and then I zoom in again...my zoom is once again at 100% zoom, automatically.

However, with JustZoom, I find that the opposite is true. If I zoom in to 200%, the next time I use my zoom keybind, it resumes the zoom at 200%. This is not expected behavior for a zoom mod, and in my opinion, it is incredibly jarring. Whenever I zoom, I want the zoom keybind to start at the same default zoom factor every time. I do not want it to remember my zoom factor from the previous zoom. I understand that not all players may feel this way, but I would greatly appreciate an option in the config to toggle a "zoom reset" feature. I have a feeling most players would prefer this by default.

**Comment by the-can-of-soup** [comment]
Yeah, I remember what you're saying used to be how it works, but the 2.0.0 update of the mod changed it. It would be cool if the feature could be added as a setting. I would downgrade to 1.0.2, but 2.0.0 fixed a bug that happened when you have different FOV than 70, and I play with 90 FOV, so i just have to deal with the zoom not resetting.

**Comment by the-can-of-soup** [comment]
Also zoom isn't working for me in 3rd person for some reason

**Comment by Keksuccino** [comment]
Fixed/Added in v2.1.0!
